pages/features.html

```diff
@@ -112,289 +142,65 @@
-          <li>{% trans "Shoutcast and Icecast broadcasting" %}</li>
-          <li>{% trans "4 Microphone Inputs" %}</li>
-          <li>{% trans "4 Auxiliary Inputs" %}</li>
-          <li>{% trans "Automatic crossfading with Auto DJ" %}</li>
-          <li>{% trans "Crates and playlists" %}</li>
-          <li>{% trans "Reads iTunes, Traktor, Banshee and Rhythmbox libraries" %}</li>
-          <li>{% trans "Pitch-independent time stretch (key lock)" %}</li>
-          <li>{% trans "Speed-neutral musical key adjustment" %}</li>
-          <li>{% trans "Vinyl emulation" %}</li>
-          <li>{% trans "Quantized loops, hot cues, and beatloops" %}</li>
-          <li>{% trans "4-deck Master Sync and auto-beatmatching" %}</li>
-          <li>{% trans "4 Sampler Decks" %}</li>
-          <li>{% trans "Wave and Ogg recording with optional MP3 support." %}</li>
-          <li>{% trans "BPM detection and estimation" %}</li>
-          <li>{% trans "Bulk BPM analysis" %}</li>
-          <li>{% trans "Multichannel soundcard support (playback and capture)" %}</li>
-          <li>{% trans "Headphone cueing with multiple soundcard support" %}</li>
-          <li>{% trans "Selectable EQ filters with adjustable shelves" %}</li>
-          <li>{% trans "Crossfader curve control" %}</li>
-          <li>{% trans "Skinnable interface with 3 scalable skins bundled" %}</li>
-          <li>{% trans "Advanced controller scripting engine" %}</li>
-          <li>{% trans "Multiple simultaneous DJ controllers" %}</li>
-          <li>{% trans "Adjustable pitch range" %}</li>
-          <li>{% trans "Ramping pitchbend controls" %}</li>
-          <li>{% trans "Multi-core CPU support" %}</li>
-          <li>{% trans "24-bit/96000 Hz playback and capture" %}</li>
-          <li>{% trans "Crystal clear bit-perfect audio" %}</li>
-          <li>{% trans "Hardware video acceleration" %}</li>
-          <li>{% trans "...and more!" %}</li>
-        </ul>
-      </div>
-      <div style="float: left; width: 25%; padding-right: 50px;">
-        <ul>
-          <p><b>{% blocktrans %}Mixxx Certified<br>Controllers{% endblocktrans %}</b></p>
-          <li>Allen &amp; Heath Xone K2</li>
-          <li>American Audio VMS 4 / 4.1</li>
-          <li>DJ TechTools MIDI Fighter Classic</li>
-          <li>Denon HS5500</li>
-          <li>Hercules DJ Console MP3 e2</li>
-          <li>Hercules DJ Console Mk2</li>
-          <li>Hercules DJ Console RMX</li>
-          <li>Hercules DJ Console RMX 2</li>
-          <li>Keith McMillen Instruments QuNeo</li>
-          <li>M-Audio X-Session Pro</li>
-          <li>Stanton SCS.1d</li>
-          <li>Stanton SCS.1m</li>
-          <li>Stanton SCS.3d</li>
-          <li>Stanton SCS.3m</li>
-          <li>Traktor Kontrol S4 MK2</li>
-          <li>Vestax VCI-400</li>
-        </ul>
-        <ul>
-          <p><b>{% blocktrans %}Community Supported<br>Controllers{% endblocktrans %}</b></p>
```

Comment:
the number one question people have is "is my controller supported."  We need to have that information very easy to locate.  Maybe a direct link to a page on the wiki?